Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/GestureBubblingTests.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+
+#if UITEST
+using NUnit.Framework;
+using Xamarin.UITest.Queries;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	// This is similar to the test for 35477, but tests all of the basic controls to make sure that they all exhibit
+	// the same behavior across all the platforms. The question is whether tapping a control inside of a frame
+	// will trigger the frame's tap gesture; for most controls it will not (the control itself absorbs the tap),
+	// but for non-interactive controls (box, frame, image, label) the gesture bubbles up to the container.
+
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.None, 00100100, "Verify that the tap gesture bubbling behavior is consistent across the platforms", PlatformAffected.All)]
+	public class GestureBubblingTests : TestNavigationPage
+	{
+		const string TargetAutomationId = "controlinsideofframe";
+		ContentPage _menu;
+
+#if UITEST
+		[Test, TestCaseSource(nameof(TestCases))]
+		public void VerifyTapBubbling(string menuItem, bool frameShouldRegisterTap)
```

Comment:
And any test for disabled controls wouldn't pass on all the platforms at this point, because they are not consistent for disabled controls. I'd rather leave that test for whoever fixes that bug.